pandas/tests/indexes/test_base.py

```diff
@@ -859,6 +859,17 @@ def test_iadd_string(self):
         index += '_x'
         assert 'a_x' in index
 
+    def test_iadd_index_name(self):
+        s = pd.Series([1, 2, 3])
+        s.index.name = 'foo'
+
+        s.index += 1
+        assert s.index.name == 'foo'
+
+    def test_reset_index_name(self):
```

Comment:
this test needs to go to pandas/tests/frame/test_alter_axes.py (with other reset_index tests)